builder/azure/common/client/config_test.go

```diff
@@ -158,6 +165,30 @@ func Test_ClientConfig_DeviceLogin(t *testing.T) {
 	}
 }
 
+func Test_ClientConfig_AzureCli(t *testing.T) {
+	getEnvOrSkip(t, "AZURE_CLI_AUTH")
```

Comment:
@nywilken from where in the tests should this be added? Can I setup something to make sure az is authenticated in the tests? I've search for the same env var in the repo for device login without finding it.